### ISSUE TITLE
Card focus indicator - Accessibility issue

### ISSUE THREAD

**Issue by venkatx5** [microsoft/BotFramework-WebChat#3894]
We are using webchat-es5.js for our Bot and as per requirement the bot should be accessibility compliant. When any card selected using keyboard/mouse, a dotted line indicator is being displayed by default which is not very clear. So we added css to make the focus indicator thicker. But when no card selected, a div is displaying on the bottom of all cards with outline.

## Screenshots
**When a card in focus**
![image](https://user-images.githubusercontent.com/11777507/118462323-2d2fb280-b71c-11eb-9074-ca679bb19529.png)

**When no card in focus**
![image](https://user-images.githubusercontent.com/11777507/118462111-f9548d00-b71b-11eb-901f-4ee05721efff.png)

## Version
webchat es5 version 4.13.0
https://cdn.botframework.com/botframework-webchat/4.13.0/webchat-es5.js
Integrating the CDN version js directly on the page using script tag (No Iframe). Desktop Chrome Browser.

```
<meta name="botframework-directlinespeech:version" content="4.13.0">
<meta name="botframework-webchat:bundle:variant" content="full-es5">
<meta name="botframework-webchat:bundle:version" content="4.13.0">
<meta name="botframework-webchat:core:version" content="4.13.0">
<meta name="botframework-webchat:ui:version" content="4.13.0">
```

## Describe the bug
As per requirement the bot should be accessibility compliant. When any card selected using keyboard/mouse, a dotted line indicator is being displayed by default. Customer requested to make the border/outline thicker so it'll be visible clearly to visually challenged users. So we added a css class for the focus indicator.

`.webchat__basic-transcript__activity-indicator--focus {
    border: 2px solid #004E9A !important;
}`

This works fine when any card is selected (in focus), but when no card is in focus, then a div with outline is being displayed at the bottom of the last card.

## Steps to reproduce
1. Deploy Web App Bot with adaptive cards.
2. Add the css  
     `.webchat__basic-transcript__activity-indicator--focus { border: 2px solid #004E9A !important; }`
3. Open the web app bot on browser.
4. Navigate the bot with keyboard.
5. Press tab to enter any message in the sendbox.
5. Observe how the focus indicator is displaying at the bottom.

## Expected behavior
The focus indicator div should be hidden or removed from DOM when no card is in focus.
Please advice if any CSS class(or other fix) can be used for the focus indication only when any message is selected.

[Bug]

**Comment by corinagum** [comment]
I'm assuming the `...--focus` class is being overwritten to always visible with your code. 

Please edit your style options instead of using CSS. https://github.com/microsoft/BotFramework-WebChat/blob/main/packages/api/src/defaultStyleOptions.ts#L199

In the future, please take a look at style options for customization before filing a bug.
